src/__test__/components/data-exploration/cell-sets-tool/CellSetsTool.test.jsx

```diff
@@ -12,11 +12,14 @@ import {
 import CellSetsTool from '../../../../components/data-exploration/cell-sets-tool/CellSetsTool';
 import CellSetOperation from '../../../../components/data-exploration/cell-sets-tool/CellSetOperation';
 import waitForComponentToPaint from '../../../../utils/tests/waitForComponentToPaint';
+import pushNotificationMessage from '../../../../utils/pushNotificationMessage';
```

Comment:
I don't think we need this, since we are not checking that it has been called.

Reply:
Removed